src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -101,8 +108,43 @@ More information about these app properties, the types they
 depend on, and how internationalized Flutter apps are typically
 structured, can be found below.
 
+### TBD: Section on adding your own localized messages?
```

Comment:
This is where I plan on introducing the new workflow. Does this seem like the right place @HansMuller?